QCs Section and Finalizing New User Manual

djcooke opened the pull request.

avarsava commented on docs/user_manual/qcs.md:
```diff
@@ -0,0 +1,41 @@
+---
+
+QCs in MISO are used to record the results of quality control checks. They can be added to samples, libraries, pools,
+and sequencing containers. It is possible to add the same QC to the same item multiple times to show a change in the
```
Unclear to me how adding the /same/ QC to an item multiple times would show a change. Knowing little about the feature, 1 QC to me seems like 1 record of a quality control check having been completed, so individual QCs should not change (that'd be rewriting history). Could the data representation here be elaborated on / is it explained elsewhere?

djcooke replied:
Reworded to clarify that multiple QCs of the same type can be added, rather than multiple of the same QCs

avarsava commented on docs/user_manual/qcs.md:
```diff
@@ -0,0 +1,41 @@
+{% assign sub = 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Adding QCs" %}
+
+A bulk input page is used for adding QCs. YOu can get there from the list page of the item type you would like to add
```
YOu -> You

djcooke replied:
I seem to do this a lot... fixed

djcooke commented:
@apmasell: added a note that partition QCs are different, which links to a new subsection of the Sequencer Runs section to explain partition QCs, and also added a Partition QC Types subsection to the Type Data section
